index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+Preferred method of contact<!DOCTYPE html>
```

Comment:
Can you please remove this change? Looks like you might have accidentally pasted it at the start of the document?